x-pack/plugins/ml/server/models/data_frame_analytics/analytics_manager.ts

```diff
@@ -47,6 +54,23 @@ export class AnalyticsManager {
     this._jobs = jobs.data_frame_analytics;
   }
 
+  public async initTransformData() {
```

Comment:
`initData` and `initTransformData`, `getInitialElementsModelRoot` and `getAnalyticsMap` can be `private`

Reply:
Updated here [`8f14114` (#162443)](https://github.com/elastic/kibana/pull/162443/commits/8f1411438f8e4e828894aaded973391e9512ef56)